### ISSUE TITLE
Sprint 28 a11y support

### ISSUE THREAD

**Issue by LynneJohnson224** [department-of-veterans-affairs/va.gov-team#79312]
## User Story
As a shared support team member, I need to understand a11y needs each MHV product team has so I can ensure we align with design system requirements and provide the best user experience possible for Veterans and other users of VA.gov. 

## Description
Any MHV teams including: Secure Messaging, Medical Records, Medications, and also related Appointments & Check-In teams may need to request shared support services for accessibility. Within a given sprint, that work should be articulated through tasks (below) and updates (comments on the ticket) as needed. 

### Ongoing tasks: 
- [x] Help teams and Bobby with design system V3 components
- [x] Prep for upcoming April research sessions (assistive tech support). Beginning sessions on 3/29.

### New tasks this sprint
- [x] Code a Codepen prototype for Medications team's refill page.
